### PR TITLE
Preparation to set up Java 11 Github Actions build

### PR DESCRIPTION
**Pull request by suztomo**
This is a preparation to setup GitHub Action for this repository. This does not run Maven build yet.

Once this gets merged to master, I should be able to run try-and-error on a subsequent pull request to run the Maven build against pull requests.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    run-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: ${{matrix.java}}
+    - run: java -version
```
Comment (suztomo) [inline .github/workflows/ci.yaml:18]: Not running real Maven build for now as Java 11 build would fail.

Comment (suztomo) [inline .github/workflows/ci.yaml:18]: This yaml file is from https://github.com/googleapis/java-cloud-bom/blob/master/.github/workflows/ci.yaml

Comment (elharo) [reply .github/workflows/ci.yaml:18]: You could take Java 11 out of the matrix.

Comment (suztomo) [reply .github/workflows/ci.yaml:18]: Nice idea. Removed 11.

**Comment by suztomo** [comment]
@elharo PTAL.

**Review by elharo — APPROVED** [review]

**Pull request merged**